manifests/grafana-dashboardDefinitions.yaml

```diff
@@ -16657,7 +16657,7 @@ items:
                           "tableColumn": "",
                           "targets": [
                               {
-                                  "expr": "sum(kubelet_running_pods{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\", instance=~\"$instance\"}) OR sum(kubelet_running_pod_count{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\", instance=~\"$instance\"})",
+                                  "expr": "sum(kubelet_running_pods{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\", instance=~\"$instance\"})",
```

Comment:
out of curiosity: why is this different than on master?

Reply:
this is a recent change that is only in kubernetes-mixins master branch and not on the release-0.6 one.

Reply:
We might have to consider bringing this to the newest mixin branch if its a fix. But we can do that outside of this PR.